packages/eslint-plugin-tslint/src/rules/config.ts

```diff
@@ -3,14 +3,10 @@ import memoize from 'lodash/memoize';
 import { Configuration, RuleSeverity } from 'tslint';
 import { CustomLinter } from '../custom-linter';
 
-// note - cannot migrate this to an import statement because it will make TSC copy the package.json to the dist folder
-// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-const version: string = require('../../package.json');
-
 const createRule = ESLintUtils.RuleCreator(
-  () =>
-    `https://github.com/typescript-eslint/typescript-eslint/blob/v${version}/packages/eslint-plugin-tslint/README.md`,
+  () => 'https://typescript-eslint.io/docs/linting/',
```

Comment:
The tslint plugin doesn't have its website pages, so this is not correct. You should update the rule creator of the _eslint plugin_ instead. Also, I'm not sure if the plan is to drop the version—I think the TS-ESLint team have plans to version the website's docs in the future?

Reply:
> update the rule creator of the _eslint plugin_

Yeah, that's the one that this issue is really about. Those are the rules folks typically use.

> version the website's docs in the future

Soft goals nobody's had time to investigate; it might be a good idea? But unversioned links will work for now.

Reply:
The "latest" version in Docusaurus will not have the version name in its URL, so the current approach is safe enough. Just want to point out that versioning is totally possible, and I assume that's the entire purpose of using two doc instances. You only want multiple doc plugin instances if they have different versioning strategies. If you don't version either of them, then it just creates troubles (i.e. unable to interlink with Markdown paths, which you are already experiencing)